src/openlcb/PolledProducer.cxxtest

```diff
@@ -61,7 +61,7 @@ protected:
         return hwState_;
     }
 
-    bool hwState_;
+    bool hwState_{false};
```

Comment:
please remove theother initialization on line 45

Reply:
Ah, I did not see that there. I'll roll back my change instead.